I have no clue why someone would override PREFIX
with something which is essentially a DESTDIR (the
INSTALL_DIR).

This caused dropbear's native Makefile to generate
bogus symlinks. Since this isn't used by the
default dropbear build it was probably never
noticed as errorneous.

One of our opensync introducing patches makes use
of inst_dropbearmulti target instead of manual
invocations of `install` program. Therefore the
--prefix and --sbindir need to be fixed so they
still do install to /bin, but maintain a sane
PREFIX.
--- a/userspace/public/apps/dropbear/Makefile
+++ b/userspace/public/apps/dropbear/Makefile
@@ -80,8 +80,8 @@ check_untar_patch_configure: sanity_chec
 	    (cd $(APP); ./configure CFLAGS="$(CFLAGS)" LDFLAGS="$(BCM_BLT32_FLAGS) -L$(INSTALL_DIR)/lib" LIBS="$(LIBS)" --host=$(TOOLCHAIN_PREFIX) \
         --target=$(TOOLCHAIN_PREFIX) \
         --disable-largefile --disable-zlib  --disable-openpty --disable-syslog  --disable-shadow  --disable-lastlog  --disable-utmp --disable-utmpx --disable-wtmp --disable-wtmpx  \
-        --prefix=$(INSTALL_DIR) \
-        --sbindir=$(INSTALL_DIR)/bin); \
+        --prefix=/ \
+        --sbindir=/bin); \
 	fi
 
 conditional_build: check_untar_patch_configure
